superbit_lensing/shear_profiles/annular_jmac.py

```diff
@@ -176,24 +453,65 @@ def compute_profile(self, outfile, overwrite=False):
             gtan_err[i] = np.std(self.gtan[annulus]) / np.sqrt(n)
             gcross_err[i] = np.std(self.gcross[annulus]) / np.sqrt(n)
 
-            print("%.3f   %d   %f   %f   %f   %f" %\
-                  (midpoint_r[i], n, gtan_mean[i], gtan_err[i], gcross_mean[i], gcross_err[i])
-                  )
-
             i += 1
 
         table = Table()
         table.add_columns(
             [counts, midpoint_r, gtan_mean, gcross_mean, gtan_err, gcross_err],
             names=[
-                'counts', 'midpoint_r', 'gtan_mean', 'gcross_mean', 'gtan_err', 'gcross_err'
+                'counts', 'midpoint_r', 'mean_gtan', 'mean_gcross', 'err_gtan', 'err_gcross'
```

Comment:
Same point as above, just flagging it so we don't forget. I'm fine either way as long as it is consistent across the repo by the time we merge.

Reply:
Great, I'll make sure the names are consistent.

Reply:
Great, I'll make sure the names are consistent.